src/Psalm/Type/Atomic/TClassString.php

```diff
@@ -51,7 +58,9 @@ public function __toString(): string
 
     public function getId(bool $nested = false): string
     {
-        return $this->getKey();
+        return ($this->is_loaded ? 'loaded-' : '')
+            . ($this->is_interface ? 'interface' : 'class')
+            . '-string' . ($this->as === 'object' ? '' : '<' . $this->as_type . '>');
     }
 
     public function getAssertionString(bool $exact = false): string
```

Comment:
every call to getAssertionString is now with $exact to true. Can we drop the param?